vscode/test/suites/language-service/notebook.test.ts

```diff
@@ -112,4 +112,43 @@ suite("Q# Notebook Tests", function suite() {
     assert.equal(location.range.start.line, 2);
     assert.equal(location.range.start.character, 10);
   });
+
+  test("Notebook defaults to unrestricted target profile", async () => {
```

Comment:
Overall looks good, but can you merge from main and try writing and running this test again? There were some new helpers added in main to wait for diagnostics, and you should use those helpers. I also fixed infrastructure so you should be able to run unit tests. merge from main first though ok? ok thx love u